parachain/pallets/verifier-lightclient/src/mock.rs

```diff
@@ -86,13 +88,31 @@ pub fn child_of_genesis_ethereum_header() -> EthereumHeader {
 	child	
 }
 
-pub fn new_tester() -> sp_io::TestExternalities {
-	let mut storage = system::GenesisConfig::default().build_storage::<MockRuntime>().unwrap();
+pub fn ethereum_header_from_file(block_num: u64) -> EthereumHeader {
+	let filename = format!("./src/testdata/{}.json", block_num);
+	serde_json::from_reader(
+		std::fs::File::open(std::path::Path::new(&filename)).unwrap()
```

Comment:
No need to use Path::new here since `str` implements `AsRef<Path>`: https://doc.rust-lang.org/std/convert/trait.AsRef.html#impl-AsRef%3CPath%3E

Basically since `str` implements `AsRef<Path>`, `File::open` knows that it can call `filename.as_ref()` to create a `Path` object.